l10n_nl_tax_statement/models/account_move_line.py

```diff
@@ -31,10 +31,52 @@ def _get_l10n_nl_vat_statement_protected_fields(self):
             "tax_tag_ids",
         ]
 
+    def check_field_is_equal(self, changed_protected_field, values):
+        self.ensure_one()
+        if self._fields.get(changed_protected_field) and self._fields[
+            changed_protected_field
+        ].type not in ["many2many", "one2many"]:
+            return self[changed_protected_field] == values[changed_protected_field]
+        # if field is X2M we must translate commands to values.
+        # the only acceptable value is [[6,0,self[changed_protected_field].ids]]
+        # wich is what the web client posts in case there is a editable X2M in form
+        # that is unchanged.
+        return (
```

Comment:
Is this tested? I get the impression, but I might be wrong, that I am missing a level, the value is a series of operations. So basically there should be one operation. Maybe nonsens, but I would expect something like:
```
operations = values[changed_protected_field]
if len operations != 1 or len(operations[0]) != 3:
    return False
return operations[0][0] == 6 and operations[0][2] == self[changed_protected_field].ids
```

Reply:
the branch is tested (green in CI)

Yes you are correct it should be [(6,0,[1.2,4])]  so it should be 
values[changed_protected_field][0][0] == 6 

Re-debugging because i remember also:
 values[changed_protected_field][2] == self[changed_protected_field].ids
being correct. will get back to you with explanation or correction.

Reply:
Done